src/style.css

```
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 0;
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+}
```

we need to clear extra margin to avoid margin collapse. So you can just do like this
```suggestion
}

.stars__star:not(:last-child) {
  margin-right: 4px
}
```